package.json

```diff
@@ -17,117 +17,90 @@
-    "@rollup/plugin-commonjs": "^18.0.0",
-    "@rollup/plugin-node-resolve": "^11.2.1",
+    "@commitlint/cli": "^17.6.6",
+    "@commitlint/config-conventional": "^17.6.6",
+    "@open-wc/testing": "^3.2.0",
+    "@rollup/plugin-node-resolve": "^15.1.0",
     "@semantic-release/changelog": "^5.0.1",
     "@semantic-release/git": "^9.0.0",
     "@semantic-release/npm": "^7.1.0",
-    "@web/dev-server": "^0.1.31",
-    "@webcomponents/webcomponentsjs": "^2.5.0",
+    "@web/dev-server": "^0.2.3",
+    "@web/test-runner": "^0.16.1",
     "autoprefixer": "^10.2.5",
-    "babel-loader": "^8.2.5",
     "compression": "^1.7.4",
     "concat": "^1.0.3",
     "copyfiles": "^2.4.1",
     "core-js": "^3.10.0",
-    "eslint": "^7.23.0",
+    "eslint": "^8.43.0",
     "eslint-plugin-lit": "^1.3.0",
-    "focus-visible": "^5.2.0",
-    "husky": "^4.3.8",
+    "husky": "^8.0.3",
     "lodash": "^4.17.21",
-    "marked": "^2.0.1",
     "nodemon": "^2.0.16",
     "npm-run-all": "^4.1.5",
     "postcss": "^8.2.9",
-    "postcss-custom-properties": "^11.0.0",
-    "postcss-discard-comments": "^4.0.2",
+    "postcss-discard-comments": "^6.0.0",
     "postcss-remove-rules": "^1.0.0",
     "postcss-selector-replace": "^1.0.2",
-    "prismjs": "^1.23.0",
     "rollup": "^2.44.0",
     "rollup-plugin-minify-html-literals": "^1.2.6",
     "rollup-plugin-serve": "^1.1.0",
-    "rollup-plugin-terser": "^7.0.2",
     "semantic-release": "^17.4.2",
-    "sinon": "^10.0.0",
-    "stylelint": "^13.12.0",
-    "stylelint-config-standard": "^21.0.0",
+    "stylelint": "^15.9.0",
+    "stylelint-config-standard": "^33.0.0",
+    "typescript": "^5.1.3",
     "wc-sass-render": "^1.2.3",
     "web-component-analyzer": "^1.1.6",
-    "webpack-merge": "^5.7.3",
     "whatwg-fetch": "^3.6.2",
     "yaml-lint": "^1.2.4"
   },
   "browserslist": [
     "last 2 Chrome versions",
     "last 2 iOS major versions",
-    "ie 11",
     "last 2 Firefox versions",
     "last 2 Edge versions",
     "last 2 Safari major versions"
   ],
   "scripts": {
-    "build": "npm run ciBuild",
-    "apiBuild": "wca analyze 'src/auro-datetime.js' --outFiles docs/api.md",
+    "build": "npm-run-all build:sass sass:render cssLint dist:js bundler postinstall build:api",
+    "build:dev:assets": "npm-run-all build:sass:component postCss:component sass:render",
+    "build:api": "wca analyze 'src/auro-badge.js' --outFiles docs/api.md",
+    "build:watch": "nodemon -e scss,js --watch src --exec npm run build:dev:assets",
     "bundler": "rollup -c",
     "bundle:test": "rollup -c -w",
-    "postinstall": "node packageScripts/postinstall.js",
+    "postinstall": "node packageScripts/postinstall.mjs",
     "cssLint": "stylelint \"./src/*.css\"",
-    "dev": "npm run sassBuild:watch",
-    "distJS": "copyfiles -u 1 -V './src/**/*.js' ./dist",
-    "sassRender": "sass-render src/*.css -t ./scripts/staticStyles-template.js",
+    "dev": "concurrently --kill-others 'npm run build:watch' 'npm run serve'",
```

Comment:
You are missing the `concurrently` package needed to run this. Have you run this locally?